stacks/patterns/index.php

```diff
@@ -0,0 +1,109 @@
+<!-- /wp:image -->
+
+<!-- wp:heading {"textAlign":"center","fontSize":"x-large"} -->
+<h2 class="wp-block-heading has-text-align-center has-x-large-font-size">convey your brand,</h2>
+<!-- /wp:heading --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"style":{"dimensions":{"minHeight":"100vh"},"position":{"type":"sticky","top":"0px"},"color":{"background":"#aaaaaa"}},"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"center","justifyContent":"center"}} -->
+<div class="wp-block-group has-background" style="background-color:#aaaaaa;min-height:100vh"><!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:image {"align":"center","id":24,"width":24,"height":24,"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image aligncenter size-full is-resized"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/currency-dollar-1.png" alt="" class="wp-image-24" width="24" height="24"/></figure>
+<!-- /wp:image -->
+
+<!-- wp:heading {"textAlign":"center","textColor":"white","fontSize":"x-large"} -->
+<h2 class="wp-block-heading has-text-align-center has-white-color has-text-color has-x-large-font-size">and most importantly, convert well.</h2>
+<!-- /wp:heading --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"style":{"dimensions":{"minHeight":"100vh"},"position":{"type":"sticky","top":"0px"},"color":{"background":"#0e0e0e"}},"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"center","justifyContent":"stretch"}} -->
+<div class="wp-block-group has-background" style="background-color:#0e0e0e;min-height:100vh"><!-- wp:cover {"url":"<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/stack.jpg","id":29,"dimRatio":50,"focalPoint":{"x":0,"y":0.5},"style":{"layout":{"selfStretch":"fill","flexSize":null}}} -->
+<div class="wp-block-cover"><span aria-hidden="true" class="wp-block-cover__background has-background-dim"></span><img class="wp-block-cover__image-background wp-image-29" alt="" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/stack.jpg" style="object-position:0% 50%" data-object-fit="cover" data-object-position="0% 50%"/><div class="wp-block-cover__inner-container"><!-- wp:group {"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"center","justifyContent":"center"}} -->
+<div class="wp-block-group"><!-- wp:heading {"textAlign":"center","textColor":"white","fontSize":"x-large"} -->
+<h2 class="wp-block-heading has-text-align-center has-white-color has-text-color has-x-large-font-size">Create your own Stacks</h2>
+<!-- /wp:heading -->
+
+<!-- wp:list {"ordered":true} -->
+<ol><!-- wp:list-item -->
+<li>Add the "stacks" pattern to a new post.</li>
```

Comment:
I noticed this text is black in the site editor, but white on the front end. Any idea why? 

<img width="1624" alt="Screenshot 2023-03-09 at 10 32 04 PM" src="https://user-images.githubusercontent.com/5375500/224165370-e7169aee-f9dc-409f-9a86-d22ddde79a9d.png">

Also it shows up with padding but maybe that's probably a bigger site wide padding issue. 

Running Gutenberg trunk.

Reply:
This is a Gutenberg issue I am investigating